numpy/testing/utils.py

```diff
@@ -903,12 +908,12 @@ def compare(x, y):
         # casting of x later.
         dtype = result_type(y, 1.)
         y = array(y, dtype=dtype, copy=False, subok=True)
-        z = abs(x-y)
+        z = abs(x - y)
 
         if not issubdtype(z.dtype, number):
             z = z.astype(float_)  # handle object arrays
 
-        return around(z, decimal) <= 10.0**(-decimal)
+        return z < 1.5 * 10.0**(-decimal)
```

Comment:
Would making this `<=` still make sense and remove the possibly stronger tests? Not sure if that makes sense or not. Anyway, looks good to me.
I would like to nudge you towards adding a test ;p. Wondering if we can somehow make it easier to understand what "decimal" means in the documentation. Also "vagaries" seems a bit difficult to me as a non-native speaker, maybe "details"?